### PR TITLE
Minor suggested edits to SSA page

### PR DESCRIPTION
**Pull request by MaxGhenis**
@jwal1024 PTAL - I'm not done yet but this also captures your #54

### CODE REVIEW

File: jb/research.md [hunk 1]
```diff
@@ -70,51 +96,115 @@ Table 3: Evidence from Sub-Saharan Africa: Cash transfers reduce child material
...
+*Note: are any of the below conditional on child attendance? If any other than NCTPP were, it'd be good to break out the UCTs into a single paragraph.*
 
+However, the evidence isn't conclusive.
+The Kenya CT-OVC and Ghana LEAP only found significant enrollment effects on children age 13 to 17 {cite}`Ghana_final` {cite}`Kenya_CT_OVC`.
+The Zimbabwe HSCT found no enrollment effects, though the authors indicate that this could be due to loss of income-based scholarships {cite}`Zimbabwe_HSCT_final`.
+Moreover, evidence from the Zambia CGP indicates that enrollment improvements did not persist: a 10 pp\*\* increase at 30 months declined to a non-significant 6 pp at 36 months.
```
Comment (jwal1024) [inline jb/research.md:162]: Needs inclusion of citation.
{cite}`Zambia_CGP_final`

File: jb/research.md [hunk 2]
```diff
@@ -70,51 +96,115 @@ Table 3: Evidence from Sub-Saharan Africa: Cash transfers reduce child material
...
+
+*TODO: Missing word in sentence above*
+
+### Child labor
+
+*TODO: Define terms*
+
+The studies considered found inconsistent evidence on the effects of cash transfers on child labor.
+While the Kenya CT-OVC found a modest reduction (-12%\*\*\*) in own-farm labor, the vast majority of studies found no effect of cash transfers on child labor incidence {cite}`CTOVC_food`.
+Indeed, the Malawi SCTP found a 0.09 pp*** increase in child labour incidence, with the majority of the increase in hazardous activities {cite}`Malawi_SCTP_final`.
```
Comment (jwal1024) [inline jb/research.md]: should be 9 pp not 0.09 pp.

File: jb/research.md (hunk 1 above)
Comment (MaxGhenis) [reply jb/research.md:162]: What would you think about putting the citations in the table once, and then referring to them by the program name (as is currently done)? They're getting referenced up to 10 times each, and it's looking a bit cluttered.

File: jb/research.md (hunk 2 above)
Comment (MaxGhenis) [reply jb/research.md]: done

File: jb/research.md (hunk 1 above)
Comment (MaxGhenis) [reply jb/research.md:162]: Per our conversation, let's do this and then add a footnote describing which studies reported which outcome metrics if any programs have multiple studies.

**Comment by MaxGhenis** [comment]
@jwal1024 will address italicized comments in a separate PR.

**Pull request merged**